frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -316,6 +321,7 @@ export class NativeQueryEditor extends Component<
     if (this.props.cancelQueryOnLeave) {
       this.props.cancelQuery?.();
     }
+    window.cancelAnimationFrame(this._focusFrame);
```

Comment:
do we really need to cancel animation frame? could you please record a glitch if we do not cancel it